Carthage installation failed

`$ carthage bootstrap --platform iOS` build failed.

```
*** Building scheme "KabuKit" in KabuKit.xcodeproj
** BUILD FAILED **


The following build commands failed:
        PhaseScriptExecution [CP]\ Check\ Pods\ Manifest.lock Build/Intermediates/KabuKit.build/Release-iphoneos/KabuKit.build/Script-4D286322F4FD3E46231E45CD.sh
(1 failure)
error: The sandbox is not in sync with the Podfile.lock. Run 'pod install' or update your CocoaPods installation.
A shell task (/usr/bin/xcrun xcodebuild -project /path/to/carthage/Carthage/Checkouts/KabuKit/KabuKit.xcodeproj -scheme KabuKit -configuration Release -sdk iphoneos ONLY_ACTIVE_ARCH=NO BITCODE_GENERATION_MODE=bitcode CODE_SIGNING_REQUIRED=NO CODE_SIGN_IDENTITY= CARTHAGE=YES clean build) failed with exit code 65:
** BUILD FAILED **
```

I found `KabuKit` has two shared scheme for iOS platform

![2017-01-13 20 33 35](https://cloud.githubusercontent.com/assets/3917725/21928772/0c6fcacc-d9d0-11e6-95e5-cc92551fc837.png)

Why `KabuKit` scheme are duplicated?
Shared `KabuKit` scheme conflicts with `KabuKit-iOS` scheme when fetching via Carthage.
To fix carthage build, delete or configure some settings shared `KabuKit` scheme.

--- a/KabuKit.xcodeproj/xcshareddata/xcschemes/KabuKitTests.xcscheme
+++ b/KabuKit.xcodeproj/xcshareddata/xcschemes/KabuKitTests.xcscheme
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <Scheme
-   LastUpgradeVersion = "0800"
+   LastUpgradeVersion = "0820"
    version = "1.3">
    <BuildAction
       parallelizeBuildables = "YES"
@@ -9,42 +9,14 @@
          <BuildActionEntry
             buildForTesting = "YES"
             buildForRunning = "YES"
-            buildForProfiling = "YES"
-            buildForArchiving = "YES"
-            buildForAnalyzing = "YES">
+            buildForProfiling = "NO"
+            buildForArchiving = "NO"
+            buildForAnalyzing = "NO">
             <BuildableReference
                BuildableIdentifier = "primary"
-               BlueprintIdentifier = "0F9289273B347440A4AD60497B2B4A1B"
-               BuildableName = "Pods_KabuKit.framework"
-               BlueprintName = "Pods-KabuKit"
-               ReferencedContainer = "container:Pods/Pods.xcodeproj">
-            </BuildableReference>
-         </BuildActionEntry>
-         <BuildActionEntry
-            buildForTesting = "YES"
-            buildForRunning = "YES"
-            buildForProfiling = "YES"
-            buildForArchiving = "YES"
-            buildForAnalyzing = "YES">
-            <BuildableReference
-               BuildableIdentifier = "primary"
-               BlueprintIdentifier = "A2E19347D12CD916835609FEC00D771C"
-               BuildableName = "Pods_KabuKitTests.framework"
-               BlueprintName = "Pods-KabuKitTests"
-               ReferencedContainer = "container:Pods/Pods.xcodeproj">
-            </BuildableReference>
-         </BuildActionEntry>
-         <BuildActionEntry
-            buildForTesting = "YES"
-            buildForRunning = "YES"
-            buildForProfiling = "YES"
-            buildForArchiving = "YES"
-            buildForAnalyzing = "YES">
-            <BuildableReference
-               BuildableIdentifier = "primary"
-               BlueprintIdentifier = "4EF5CCE51DE30A0C00683D1E"
-               BuildableName = "KabuKit.framework"
-               BlueprintName = "KabuKit"
+               BlueprintIdentifier = "4EF5CCEE1DE30A0C00683D1E"
+               BuildableName = "KabuKitTests.xctest"
+               BlueprintName = "KabuKitTests"
                ReferencedContainer = "container:KabuKit.xcodeproj">
             </BuildableReference>
          </BuildActionEntry>
@@ -54,8 +26,7 @@
       buildConfiguration = "Debug"
       selectedDebuggerIdentifier = "Xcode.DebuggerFoundation.Debugger.LLDB"
       selectedLauncherIdentifier = "Xcode.DebuggerFoundation.Launcher.LLDB"
-      shouldUseLaunchSchemeArgsEnv = "YES"
-      codeCoverageEnabled = "YES">
+      shouldUseLaunchSchemeArgsEnv = "YES">
       <Testables>
          <TestableReference
             skipped = "NO">
@@ -68,15 +39,6 @@
             </BuildableReference>
          </TestableReference>
       </Testables>
-      <MacroExpansion>
-         <BuildableReference
-            BuildableIdentifier = "primary"
-            BlueprintIdentifier = "4EF5CCE51DE30A0C00683D1E"
-            BuildableName = "KabuKit.framework"
-            BlueprintName = "KabuKit"
-            ReferencedContainer = "container:KabuKit.xcodeproj">
-         </BuildableReference>
-      </MacroExpansion>
       <AdditionalOptions>
       </AdditionalOptions>
    </TestAction>
@@ -93,9 +55,9 @@
       <MacroExpansion>
          <BuildableReference
             BuildableIdentifier = "primary"
-            BlueprintIdentifier = "4EF5CCE51DE30A0C00683D1E"
-            BuildableName = "KabuKit.framework"
-            BlueprintName = "KabuKit"
+            BlueprintIdentifier = "4EF5CCEE1DE30A0C00683D1E"
+            BuildableName = "KabuKitTests.xctest"
+            BlueprintName = "KabuKitTests"
             ReferencedContainer = "container:KabuKit.xcodeproj">
          </BuildableReference>
       </MacroExpansion>
@@ -111,9 +73,9 @@
       <MacroExpansion>
          <BuildableReference
             BuildableIdentifier = "primary"
-            BlueprintIdentifier = "4EF5CCE51DE30A0C00683D1E"
-            BuildableName = "KabuKit.framework"
-            BlueprintName = "KabuKit"
+            BlueprintIdentifier = "4EF5CCEE1DE30A0C00683D1E"
+            BuildableName = "KabuKitTests.xctest"
+            BlueprintName = "KabuKitTests"
             ReferencedContainer = "container:KabuKit.xcodeproj">
          </BuildableReference>
       </MacroExpansion>

--- a/fastlane/Fastfile
+++ b/fastlane/Fastfile
@@ -10,7 +10,7 @@ platform :ios do
   desc "Runs tests as iOS"
   lane :test do
     scan(
-         scheme: "KabuKit",
+         scheme: "KabuKitTests",
          devices: ["iPhone 6s"],
          clean: true,
          sdk: "iphonesimulator",

